src/test/java/Homework16.java

```diff
@@ -0,0 +1,24 @@
+import org.checkerframework.common.value.qual.StaticallyExecutable;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest{
+    @Test
+    public void Navigation(){
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id=hel']"));
+        registrationLink.click();
+        String registrationUrl ="https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(),registrationUrl);
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221504455-04a8c8ec-5c25-4f4a-a725-aaf726b0ea1a.png)

**Hint:**   `WebElement registrationLink = driver.findElement(By.id("hel"));`

Reply:
ok fixed it and I'm showing passed this time.